drivers/ethernet/eth_adin2111.c

```diff
@@ -935,13 +1019,14 @@ static int adin2111_filter_multicast(const struct device *dev)
 {
 	const struct adin2111_config *cfg = dev->config;
 	bool is_adin2111 = (cfg->id == ADIN2111_MAC);
-	uint8_t mm[6] = {BIT(0), 0U,  0U, 0U, 0U, 0U};
+	uint8_t mm[6] = {BIT(0), 0U, 0x5EU, 0xFFU, 0xFFU, 0xFFU};
+	uint8_t mmask[6] = {0xFFU, 0xFFU, 0xFFU, 0x00U, 0x00U, 0x00U};
```

Comment:
That prevents reception of LLDP, PTP, and others from the [01, 80, C2] (IEEE 802 group) required by some applications.

If there is a desire to limit multicasts to [01, 00, 5E], then it must be configurable.

Reply:
@GeorgeCGV oh ok. Non really clear to me, may be wrong, but from testing, multicast 0x01 0x00 0x00 0x00 0x00 0x00 allows broadcast packets to be received too, so broadcast filter comes useless.

Reply:
Indeed, currently used mask should be corrected to `0xFF, 0x0, 0x0, 0x0, 0x0, 0x0` instead of `0x1, 0x0, 0x0, 0x0, 0x0, 0x0`.
That should fix broadcast and multicast filters.

However, we don't want to limit the "kind" of the multicast. Therefore, the mac stays as `0x1, 0x0, 0x0, 0x0, 0x0, 0x0`.

Reply:
ok, done, thanks